src/js/common.js

```diff
@@ -314,3 +314,15 @@ module.exports.setRadioSelectors = () => {
     }
   }
 }
+
+global.renderChecksum = function(link) {
+  var settings = {
+    "url": `https://cors-anywhere.herokuapp.com/${link}`,
```

Comment:
From https://github.com/Rob--W/cors-anywhere/
> A public demo of CORS Anywhere is available at https://cors-anywhere.herokuapp.com. This server is only provided so that you can easily and quickly try out CORS Anywhere

Reply:
thanks @BethGriggs I'm going to have to get it to work with cors anyway so will need to investigate fixing the original issue and not hacking with https://cors-anywhere.herokuapp.com.